### PR TITLE
Bumps in harmonic : ci_matching_branch/bump_harmonic_sdformat14

### PR DESCRIPTION
**Pull request by azeey**
See https://github.com/gazebo-tooling/release-tools/issues/976

### CODE REVIEW

File: jammy/debian/changelog [hunk 1]
```diff
@@ -1,71 +0,0 @@
-sdformat13 (13.5.0-1~jammy) jammy; urgency=medium
```
Comment (scpeters) [inline jammy/debian/changelog:1]: I think there needs to be at least one entry in a changelog for our deb-builders to work:

* https://github.com/gazebo-release/gz-sensors8-release/blob/main/jammy/debian/changelog

**Comment by scpeters** [comment]
this would be a little easier to review if the `rm -rf focal` was in a separate commit than the other changes, but I can manage as is

File: ubuntu/debian/tests/build [hunk 2]
```diff
@@ -32,11 +32,11 @@ cmake_minimum_required(VERSION 3.5)
 
 project(sdf_test VERSION 1.0.0)
 
-find_package(sdformat13 13.0.0 REQUIRED)
+find_package(sdformat14 13.0.0 REQUIRED)
```
Comment (scpeters) [inline ubuntu/debian/tests/build]:
```suggestion
find_package(sdformat14 14.0.0 REQUIRED)
```

File: jammy/debian/changelog (hunk 1 above)
Comment (azeey) [reply jammy/debian/changelog:1]: Fixed in https://github.com/gazebo-release/sdformat14-release/pull/1/commits/fdb76cb117183737c9d111e3d0935cf1f65eb669

File: ubuntu/debian/tests/build (hunk 2 above)
Comment (azeey) [reply ubuntu/debian/tests/build]: Fixed in https://github.com/gazebo-release/sdformat14-release/pull/1/commits/fdb76cb117183737c9d111e3d0935cf1f65eb669

**Comment by azeey** [comment]
> this would be a little easier to review if the `rm -rf focal` was in a separate commit than the other changes, but I can manage as is

Yeah, sorry about that. I've separated the changes in two commits and addressed your feedback in the third.

File: ubuntu/debian/compat [hunk 3]
```diff
@@ -1 +1 @@
-9
+11
```
Comment (iche033) [inline ubuntu/debian/compat]: checking to see if this is intended? Looking at a few other release repos, the debhelper compatibility levels are set to 9.

Comment (azeey) [reply ubuntu/debian/compat]: I made the change after seeing https://github.com/gazebo-release/sdformat13-release/pull/18.

Comment (iche033) [reply ubuntu/debian/compat]: ah ok I see

**Review by iche033 — APPROVED** [review]

File: ubuntu/debian/compat (hunk 3 above)
Comment (iche033) [reply ubuntu/debian/compat]: maybe we also need to change this from 13 to 11: https://github.com/gazebo-release/sdformat14-release/blob/ci_matching_branch/bump_harmonic_sdformat14/jammy/debian/compat

Comment (azeey) [reply ubuntu/debian/compat]: Oh interesting, I think we actually need 13, so I'll update this file and make the jammy one symlink to this. I found https://manpages.ubuntu.com/manpages/jammy/en/man7/debhelper.7.html#compatibility%20levels which states 13 is the recommended level for Jammy. For Focal, https://manpages.ubuntu.com/manpages/focal/en/man7/debhelper.7.html#compatibility%20levels says 12 is recommended and actually says not to use 11.

Comment (iche033) [reply ubuntu/debian/compat]: sounds good!

**Comment by azeey** [comment]
Test build: [Build Status](https://build.osrfoundation.org/view/ign-harmonic/job/sdformat14-debbuilder/1/)

**Pull request merged**